src/Compilers/CSharp/Portable/Symbols/Metadata/PE/MetadataDecoder.cs

```diff
@@ -407,7 +407,8 @@ public static bool IsOrClosedOverATypeFromAssemblies(TypeSymbol symbol, Immutabl
 
                         // Let's use a trick. To make sure the kind is the same, make sure
                         // base type is the same.
```

Comment:
Is this comment accurate now? Appears we are comparing special types now vs. base types as before.

Reply:
Yes, it is accurate, I believe. This is a form of comparison